routes/middlewares/uploadVideo.js

```diff
@@ -20,9 +20,11 @@ const s3VideoUploader = multerS3({
   },
 });
 
-exports.uploadVideo = multer({
+const uploadVideo = multer({
```

Comment:
AWS 서버로 업로드하는 함수니까 아래와 같이 이름을 수정하는건 어떨까요?
```suggestion
const uploadVideoToAWS = multer({
```

Reply:
네, 좋은 의견 감사합니다. 
다소 오해의 소지가 있을 수 있을 것 같아 말씀 주신 대로 수정 하겠습니다.